docs/sources/enterprise/access-control/roles.md

```diff
@@ -48,6 +48,14 @@ Roles with names prefixed by `fixed:` are fixed roles created by Grafana and can
 
 A role’s display name is human friendly text that is displayed in the UI. When you create a display name for a role, use up to 190 ASCII-based characters. For fixed roles, the display name is shown as specified. If the display name has not been set the display name replace any `:` (a colon) with ` ` (a space).
 
+### Display name
+
+A role's display name is intended as a human friendly identifier for the role, helping users understand the purpose of a role. The display name of the role is displayed in the role picker in the UI.
```

Comment:
```suggestion
A role’s display name is a human-friendly identifier for the role, so that users more easily understand the purpose of a role. You can see the display name in the role picker in the UI.
```